example/android/app/src/main/jni/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+cmake_minimum_required(VERSION 3.13)
+# Define the library name here.
+project(example_appmodules)
+# This file includes all the necessary to let you build your application with the New Architecture.
+include(${REACT_ANDROID_DIR}/cmake-utils/ReactNative-application.cmake)
```

Comment:
no newline :O

Reply:
Added in https://github.com/software-mansion/react-native-gesture-handler/pull/2209/commits/062af7ce45ef82ada9cd4c96275746d8cb2dff1f